docs/debugging.md

```diff
@@ -56,3 +56,10 @@ You *can't*:
 * Perform most other ordinary debugging scenarios.
 
 Development of further debugging scenarios is an on-going focus of the engineering team.
+
+## Troubleshooting tips
+
+If you are running into errors, the following tips may help.
+
+* Use the HTTP URL for your app instead of the HTTPS to debug.
```

Comment:
I can remove this for the time being then. Using my Surface Pro, with the latest Chrome, I can't get it to work with https://localhost:5001 using the sample dotnet new blazor project. This is with .NET 2.1.403. HTTP works just fine though. 

The debugging window gives the "Debugging connection was closed. Reason WebSock disconnected" error. The console says the WebSocket connection closed before receiving a handshake response.

My random guess was this had to do with some type of HTTPS issue since the debugging window is not HTTPS but I'm just throwing paint at a wall with this guess.